### DISCUSSION TITLE
Installazione del registro

### DISCUSSION THREAD

**Discussion by giadaschiavone** [iisgiua/giuaschool#488] [Richieste di aiuto]
Buongiorno, vorrei richiedere un video oppure una spiegazione dettagliata in cui si illustra l'installazione del registro su un sito ad hosting abilitato WordPress.

Cordiali saluti,
prof.ssa Giada Schiavone

**Comment by trinko** [comment]
Ciao, l'uso di WordPress non è compatibile con l'applicazione del registro elettronico. Devi usare un hosting abilitato alla gestione di un sito web generico, in modo da poter effettuare tutta la gestione.
L'installazione consiste nel caricare l'ultima versione sul server e poi seguire le istruzioni:

[vedi la documentazione](https://iisgiua.github.io/giuaschool-docs/install-zip.html)

**Reply by giadaschiavone** [reply]
Ciao, più esattamente userei Altervista e non saprei poi come muovermi, non so come installare seguendo le sue istruzioni, per questo ho chiesto un video.

Grazie mille,
prof.ssa Giada Schiavone.

**Comment by giadaschiavone** [comment]
Ciao, più esattamente userei Altervista e non saprei poi come muovermi, non so come installare seguendo le sue istruzioni, per questo ho chiesto un video.

Grazie mille,
prof.ssa Giada Schiavone.

**Comment by trinko** [comment]
Ci dovrebbe essere qualcuno che offre supporto per l'installazione. 
Prova a scrivere un'email al seguente indirizzo, che è stato più volte citato su questo forum:

giuaschoolregistro@gmail.com
